### PR TITLE
chore(pnpm): migrate to pnpm 7 lockfile + 7.15.0

### PR DESCRIPTION
**Pull request by admmasters**
Switching to pnpm7 to use the latest features.

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.3
+lockfileVersion: 5.4
```
Comment (admmasters) [inline pnpm-lock.yaml:1]: weirdly this was specifying a lockfile format of **5.3** even though **5.4** is the one used within pnpm7

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Setup pnpm
         uses: pnpm/action-setup@v2.0.1
         with:
-          version: 7.5.0
+          version: 7.15.0
```
Comment (admmasters) [inline .github/workflows/ci.yaml:30]: Setting to the current 7.15.0

File: netlify.toml [hunk 3]
```diff
@@ -1,7 +1,4 @@
-# Workarround netlify with pnpm
-# See: https://github.com/netlify/build/issues/1633
-
 [build]
-  command = "./etc/netlify/netlify.sh"
+  command = "pnpm run build"
```
Comment (admmasters) [inline netlify.toml:2]: 🕺🏽
https://www.netlify.com/blog/how-to-use-pnpm-with-netlify-build/

File: netlify.toml [hunk 4]
```diff
@@ -1,7 +1,4 @@
...
 [build.environment]
   NPM_FLAGS="--prefix=./etc/netlify/empty-project/"
```
Comment (admmasters) [inline netlify.toml:4]: @poteirard not sure if we need this anymore? 🤔

**Review by poteirard — APPROVED** [review]
Great work ❤️

**Pull request merged**